### ISSUE TITLE
Allow horizontal resizing but not full screen toggling of TinyMCE editor

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/janeway#4034]
## Problem

We have currently allowed the fullscreen feature on the TinyMCE editor. It has some accessibility issues when combined with our themes. The Escape key does not allow you to exit the full-screen view on Firefox on the OLH theme or the Hourglass theme. And on the Hourglass theme, the sticky nav bar covers up the visual interface for exiting full screen. As such it is a bit of a trap.

### OLH theme - no escape key

![image](https://github.com/BirkbeckCTP/janeway/assets/47217308/7f037575-ab16-4401-bc0e-3fbaa23c7438)

### Hourglass theme - no Escape key or button

![image](https://github.com/BirkbeckCTP/janeway/assets/47217308/81b75a9a-b1fc-4bd4-9db2-54d0ea6f2999)

## Proposed solution
Rather than enabling the fullscreen toggle, use the resize options to allow both horizontal and vertical resizing.

We currently have an invalid setting for "resize":

https://github.com/BirkbeckCTP/janeway/blob/d52b7719bf2bad26a9779246cf3305f0aa753ea2/src/core/janeway_global_settings.py#L680

![image](https://github.com/BirkbeckCTP/janeway/assets/47217308/7aa8a8ef-76bb-4080-93b4-c282191c710f)

This should be True, False, or 'both'. We can use both to allow resizing in both directions, like this:

![image](https://github.com/BirkbeckCTP/janeway/assets/47217308/3a07bd17-7302-42ed-8680-69938399b14c)

## Alternatives
I tried debugging the Escape key event and the covered-up nav but could not do so in the time we have.

## Additional context
[TinyMCE docs: Controlling user editor resizing](https://www.tiny.cloud/docs/tinymce/6/editor-size-options/#controlling-user-editor-resizing)
